app/renderer/js/main.js

```diff
@@ -530,6 +533,29 @@ class ServerManagerView {
 							}
 						});
 					}
+				},
+				{
+					label: (mutedOrganizations[url] ? 'Unmute' : 'Mute') + ' organization',
```

Comment:
Note that stuff like this is not going to work with translations

Reply:
I'm sorry, I didn't think about that. I'll need to have a look at how translations are currently handled. Will get to it.